src/Compute/v2/Models/Limit.php

```diff
@@ -0,0 +1,21 @@
+<?php declare (strict_types = 1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenCloud\Common\Resource\OperatorResource;
+
+/**
+ * Represents a Compute v2 Limit
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class Limit extends OperatorResource
+{
+    /** @var string */
+    public $rate;
+
+    /** @var object */
+    public $absolute;
```

Comment:
hmm. instead of an object, can we load all of the object variables as properties for this class instead? so it'll be: 
```
"maxServerMeta": 128,
            "maxPersonality": 5,
            "totalServerGroupsUsed": 0,
            "maxImageMeta": 128,
            "maxPersonalitySize": 10240,
            "maxTotalKeypairs": 100,
            "maxSecurityGroupRules": 20,
            "maxServerGroups": 10,
            "totalCoresUsed": 1,
            "totalRAMUsed": 2048,
            "totalInstancesUsed": 1,
            "maxSecurityGroups": 10,
            "totalFloatingIpsUsed": 0,
            "maxTotalCores": 20,
            "maxServerGroupMembers": 10,
            "maxTotalFloatingIps": 10,
            "totalSecurityGroupsUsed": 1,
            "maxTotalInstances": 10,
            "maxTotalRAMSize": 51200
```

we can probably do it with `populateFromArray`